src/borg/archiver.py

```diff
@@ -2234,6 +2234,11 @@ def do_break_lock(self, args, repository):
         /path/to/repo /home/user`` will store all files as
         `home/user/.../file.ext`.
 
+        A folder exclusion pattern can end either with or without a slash ('/').
```

Comment:
s/folder/directory/ please.

i checked the patterns help, it nowhere else uses "folder", but always talks about "directory".

The paths mention here are usually included in backticks, like:
```
... without a slash (`/`).
```

Maybe check if that renders ok here, too.

Reply:
Should it actually be backticks? I don't mind changing it but I've used single quotes here because that's how it is in several instances below (also for ':', '*', '?'). It's not consistent as there are also a few instances of backticks for single characters, but less.

Ok for folder/directory.

Reply:
Hmm, right, when talking about single chars, it uses normal ticks. OK, so we keep that one as is.